### ISSUE TITLE
sudo modprobe 88x2bu not working

### ISSUE THREAD

**Issue by digo-ca** [cilynx/rtl88x2bu#90]
**command**: _sudo modprobe 88xbu_

**the execution never ends**

**uname -a:**
> _Linux U2COIN48402 5.4.0-51-generic #56-Ubuntu SMP Mon Oct 5 14:28:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux_

**Comment by hsa1280** [comment]
I have the same issue, also the WiFi icon is not showing up on Ubuntu, maybe it is caused by this issue.

**Comment by MaxG87** [comment]
Is this issue still relevant? Unfortunately, I do not know how to help here.

**Comment by digo-ca** [comment]
Hello, I don't know what caused mine to work.  I believe the problem occurred when I performed the entire procedure with the wireless adapter connected.  I'm not sure if that was it, but when I did the procedure and only after I connected the adapter, it worked.

**Comment by digo-ca** [comment]
But for it is no longer relevant, for me it could end without problems @MaxG87.
